x/gmp/genesis.go

```diff
@@ -0,0 +1,35 @@
+package gmp
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/sagaxyz/ssc/x/gmp/keeper"
+	"github.com/sagaxyz/ssc/x/gmp/types"
+)
+
+// InitGenesis initializes the module's state from a provided genesis state.
+func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
+    // this line is used by starport scaffolding # genesis/module/init
+k.SetPort(ctx, genState.PortId)
```

Comment:
Incorrect indentation. There are also other formatting errors across the entire module. Please run gofmt on every file in this PR (except generated files).